app/views/orders/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="container">
+  <h1> Order Confirmation </h1>
+  <% @order.line_items.each do |line_item| %>
+    <p> Title: <%= line_item.book.title %> </p>
+    <p> Author: <%= line_item.book.author %> </p>
+    <p> Published On: <%= line_item.book.published_date %> </p>
+    <p> Price: <%= line_item.book.price %> </p>
+    <p> Quantity: <%= line_item.quantity %> </p>
+    <p> Total Price: <%= line_item.total_price %> </p>
+  <% end %>
+  <h3> Total: <%= @order.total %> </h3>
```

Comment:
Yes - will do some formatting with decorators in a future PR. Right now, it's cents everywhere throughout the app.